.gitignore

```diff
@@ -33,3 +33,4 @@ tools/conf.ini
 tools/custom_provision
 api/bots/john/assets/var/database.db
 static/locale/en
+.mypy_cache/
```

Comment:
We use `var/mypy-cache` for mypy, so I think this comes from running `mypy` not through `tools/run-mypy`?  I guess I'm not sure if we should include it.